esmvaltool/preprocessor/_download.py

```diff
@@ -50,7 +51,7 @@ def synda_search(variable):
     return files
 
 
-def synda_download(synda_name, dest_folder):
+def synda_download(synda_name, dest_folder, variable, rootpath, drs):
```

Comment:
The new extra arguments do not appear to be used for anything, can you remove them again?